doc/uploads/plantuml/ara_sequence_diagram.puml

```diff
@@ -0,0 +1,18 @@
+@startuml
+group upload zip (ExecutionRessource)
+  ARA <- CI : sending ZIP
+  note left: calling the resource /upload
+  group analyzing report (ExecutionService)
+    ARA -> ARA : get report details (executions) from zip
+    ARA -> ARA : save details in the server ARA is deployed on
+    note right: saved in the ara-server/path/{{project}}/{{branch}}/{{cycle}}
+    ARA -> ARA: creating indexed builds for each executions extracted from the zip
+    group searching index (ExecutionCrawlerService)
+      group fetching the executions (FetcherService)
+        ARA -> CI: getting the executions
```

Comment:
Wrong, ARA is autonomous and not able to contact the CI.

Executions are retrieved directly from the ZIP file like this: 
```
executions.zip
   1234567890/                 <-- first execution
         .....
    2345567891/                <-- second execution
         ......
```

Reply:
done